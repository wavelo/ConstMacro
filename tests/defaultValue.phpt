<?php

include 'bootstrap.php';


assertTemplate('defaultValue');
