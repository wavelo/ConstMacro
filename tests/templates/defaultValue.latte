
{const [title:='Title'] = $props /}
{$title}

{const [title:=''] = $props /}
{$title}

{const [title:=1] = $props /}
{$title}

{const [title:] = $props /}
{$title?:'Title2'}

{const [title:=0] = $props /}
{$title}

{const [title:=NULL] = $props /}
{$title===NULL ? 'NULL' : '0'}

{const [title:=] = $props /}
{$title===NULL ? 'NULL' : '0'}

{const [title:] = $props /}
{$title===NULL ? 'NULL' : '0'}

{const [title:=0] = $props /}
{$title===NULL ? 'NULL' : '0'}
