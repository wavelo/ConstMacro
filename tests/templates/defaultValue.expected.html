
Title

1
Title2
0
NULL
NULL
NULL
0
